src/Orleans.Runtime/Hosting/DefaultSiloServices.cs

```diff
@@ -274,8 +274,21 @@ internal static void AddDefaultServices(IApplicationPartManager applicationPartM
             services.AddTransient<IConfigurationValidator, ClusterOptionsValidator>();
             services.AddTransient<IConfigurationValidator, SiloClusteringValidator>();
 
-            // Disable hosted client by default.
-            services.TryAddSingleton<IHostedClient, DisabledHostedClient>();
+            // Enable hosted client.
+            services.AddSingleton<HostedClient>();
+            services.AddFromExisting<IHostedClient, HostedClient>();
```

Comment:
So users can no longer provider their own impl of IHostedClient as default?  I'm cool with that, just want to make sure we're aware of this change.  Users can still register IHostedClient implementaions by name as a named service if they need to use a custom impl or connect to multiple clusters (I think..?)

Reply:
`IHostedClient` is intentionally internal. It doesn't even need an interface anymore since it's always enabled and we only have one valid implementation of it. There's no pressing reason to remove the interface yet, though.

To connect to a remote cluster, the external client would currently have to be used.